docs/modules/operation/pages/deep-dive/admin/configuration/scv-configuration.adoc

```diff
@@ -7,6 +7,20 @@ It stores them in the Java keystore (JKS) format.
 
 The secure credentials vault is enabled by default.
 
+== Configure keystore type and file name
```

Comment:
By default, the Secure Credentials Vault in OpenNMS uses a JCEKS keystore with the file name scv.jce. OpenNMS supports two keystore types: JCEKS and PKCS12. You can customize both the keystore type and file name by setting the appropriate properties in: